rest_framework-stubs/decorators.pyi

```diff
@@ -7,23 +9,22 @@ from rest_framework.renderers import BaseRenderer
 from rest_framework.schemas.inspectors import ViewInspector
 from rest_framework.throttling import BaseThrottle
 
-_ViewCallable = Callable
-_ViewDecorator = Callable[[Callable], _ViewCallable]
+_VIEW = TypeVar("_VIEW", bound=Callable[..., HttpResponseBase])
```

Comment:
Note that this doesn't work if I make `_ViewDecorator` an alias for `Callable[[_VIEW], _VIEW]`. It only works if the `Callable[]` is directly the function's return type.

I used the variable name `_VIEW` instead of `_ViewCallable`, because TypeVar variables are usually in allcaps.